drivers/mtd/Kconfig

```diff
@@ -382,10 +384,10 @@ config AT24XX_SIZE
 	int "AT24xx size (Kbit)"
 	default 64
 	---help---
-		This is the XX in the AT24Cxx part number.  For example, if you have a
-		AT 24C512, then the correct value is 512.  This value is also the capacity
-		of the part in kilobits.  For example, the 24C512 supports 512 Kbits or
-		512 /8 = 64 KiB.
+		This is the XX in the AT24Cxx part number.  For example, if you have
+		an AT24C512, then the correct value is 512.
+		This value is also the capacity of the part in kilobits.
+		For example, the 24C512 supports 512 Kbits or 512 /8 = 64 KiB.
```

Comment:
@TimJTi I think there is an error in this original comment: it says the size is in kilobits, but of course it is in KiB (Kilo Byte). Normally flashes memory are described in Kb and we need to divide by 8 to get the value in KiB. Please fix this comment.

Reply:
@acassis I think it is right? An AT24C512 is the capacity is in kbit - 512kbit, or 64KiB as stated. Or can I "not see for looking"? And this comment relates to AT24XX not AT25EE that I have added?

Reply:
@TimJTi This is the point: AT24C512 is 512Kbit but the default value is set to 64Kbit according with the prompt message: "AT24xx size (Kbit)", understood now?

The prompt should be: "AT24xx size (in KiB)" and the Help message also need to be updated, it says that the entered value is in Kilo Bit, but actually it is in KiB.

Reply:
I disagree; or I am being very stupid!

The value you enter in Kconfig is in kbits as it says. The default is 64 kbits.

The example in the help is for an AT24C512, 512bits, which is 64KiB.

It might be inconsistent that the help example doesn't match the default, but it is not wrong as such.

Would you like me to change the example - in part of the Kconfig that is unrelated to this PR - to be for an AT24C64, 64kbit and therefore 8KiB?

Reply:
Ok, I think I was fooled by the comment "This value is also the capacity..." and "For example 512Kbits... = 64 KiB"

I was assuming the the XX = 64 was the 64 KiB, because it is mixing two information with same value:

```
		This is the XX in the AT24Cxx part number.  For example, if you have
		an AT24C512, then the correct value is 512.
		This value is also the capacity of the part in kilobits.
		For example, the 24C512 supports 512 Kbits or 512 /8 = 64 KiB.
```

If it was the default value of 64 (AT24C64), the example could be clear:
```
For example, the 24C64 supports 64 Kbits or 64 /8 = 8 KiB.
```

I think this Help is clear, but of course, the example with same 64 value could fool other people :-)

Reply:
I have edited this on behalf of whoever created this driver so the default value is 64